Neos.Flow/Classes/Mvc/Routing/Router.php

```diff
@@ -151,29 +119,6 @@ public function getLastMatchedRoute()
         return $this->lastMatchedRoute;
     }
 
-    /**
-     * Returns a list of configured routes
-     *
-     * @return array
-     */
-    public function getRoutes()
-    {
-        $this->createRoutesFromConfiguration();
-        return $this->routes;
-    }
-
-    /**
-     * Manually adds a route to the beginning of the configured routes
-     *
-     * @param Route $route
-     * @return void
-     */
-    public function addRoute(Route $route)
```

Comment:
this can be considered breaking

Reply:
Nope - because it was never a part of the RouterInterface